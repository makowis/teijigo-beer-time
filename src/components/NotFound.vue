<template>
  <div class="not-found">
    <h1>404 NOT FOUND</h1>
    <p>このページは存在しません。</p>
    <img src="../assets/nomikai_salaryman.png" alt="飲み会のイラストです。ビール飲みましょう">

  </div>
</template>

<script lang="ts">
import Vue from 'vue';

export default Vue.extend({
  name: 'not-found',
});
</script>

<style scoped>
</style>
