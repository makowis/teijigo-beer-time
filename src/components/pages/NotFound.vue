<template>
  <tbt-default>
    <div class="not-found">
      <tbt-h1>404 NOT FOUND</tbt-h1>
      <p>このページは存在しません。</p>
      <img
        src="@/assets/nomikai_salaryman.png"
        alt="飲み会のイラストです。ビール飲みましょう"
      />
    </div>
  </tbt-default>
</template>

<script lang="ts">
import Vue from 'vue';
import Component from 'vue-class-component';
import TbtDefault from '@/components/templates/TbtDefault.vue';
import TbtH1 from '@/components/atoms/TbtH1.vue';

@Component({
  components: {
    TbtDefault,
    TbtH1,
  },
})
export default class NotFound extends Vue {}
</script>

<style scoped></style>
