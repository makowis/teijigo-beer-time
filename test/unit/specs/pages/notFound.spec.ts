import { mount, RouterLinkStub } from '@vue/test-utils'
import NotFound from '@/pages/notFound.vue';

describe('index.vue', () => {
  const vm = mount(NotFound, {
    stubs: {
      NuxtLink: RouterLinkStub
    }
  }).vm;  

  it('should match the snapshot', () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
