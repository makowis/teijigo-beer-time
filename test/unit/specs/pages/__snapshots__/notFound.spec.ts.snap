// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue should match the snapshot 1`] = `
<div
  class="not-found"
>
  <h1
    class="h1"
  >
    404 NOT FOUND
  </h1>
   
  <p>
    このページは存在しません。
  </p>
   
  <img
    alt="飲み会のイラストです。ビール飲みましょう"
    src="@/assets/nomikai_salaryman.png"
  />
</div>
`;
